docs/extensibility-http.md

```diff
@@ -0,0 +1,483 @@
+# Extensibility: HTTP-based Devices
+
+See [Extensibility: HTTP protocol](https://github.com/deislabs/akri/issues/85)
+
+This documentation will be completed once the code is working.
```

Comment:
i think the end-goal should be that the contents of this file will replace the `nessie` contents in the docs/extensibility.md in `main` ... and that this file will not be present in this PR.

Reply:
I messed up cleaning up the other PR on `protocol-http`.

Should I delete `docs/extensibility.md` and mv `doc/extensibility-http.md` onto it here?

Reply:
i was thinking more of a merge ... that you'd move the contents of `docs/extensibility-http.md` into `docs/extensibility.md`, but keep extensibility.md's intro (**Extensibility**) and conclusion (**Contributing your Protocol Implementation back to Akri**)